src/AppKit/Enums.cs

```diff
@@ -2735,4 +2747,106 @@ public enum NSScrubberAlignment : nint
 		Center
 	}
 
+	[Mac (10,13)]
+	public enum NSFontError : int {
+
+		AssetDownloadError = 66304,  // Generic code when font asset download error occurs.
+		ErrorMinimum = 66304,  // Generic code when font asset download error occurs.
+		ErrorMaximum = 66335,  // Generic code when font asset download error occurs.
+	}
+
+	[Mac (10,13)]
+	[Native]
+	public enum NSAccessibilityAnnotationPosition : nint
+	{
+		FullRange,
+		Start,
+		End
```

Comment:
minor: better end with a comma (,) so adding future values results in smaller diffs